### PR TITLE
backend-plugin-api: inlined LogMeta type

### PR DESCRIPTION
**Pull request by Rugvip**
## Hey, I just made a Pull Request!

🧹 

Removing this one for now

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: packages/backend-test-utils/api-report.md [hunk 1]
```diff
@@ -111,14 +111,7 @@ export namespace mockServices {
...
     export type Options = {
-      levels:
-        | boolean
-        | {
-            error: boolean;
-            warn: boolean;
-            info: boolean;
-            debug: boolean;
-          };
+      level?: 'none' | 'error' | 'warn' | 'info' | 'debug';
```
Comment (benjdlambert) [inline packages/backend-test-utils/api-report.md:114]: Hmm this is a breaking change right?

**Review by benjdlambert — COMMENTED** [review]
Looks good, just noticed this?

File: packages/backend-test-utils/api-report.md (hunk 1 above)
Comment (Rugvip) [reply packages/backend-test-utils/api-report.md:114]: Yep but it's an alpha export

Comment (benjdlambert) [reply packages/backend-test-utils/api-report.md:114]: Yep - just thought about the changeset and calling it out as to what's different?

File: .changeset/hip-tables-prove.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-app-api': patch
```
Comment (Rugvip) [inline .changeset/hip-tables-prove.md:2]:
```suggestion
'@backstage/backend-test-utils': patch
```

Oops 😅

**Review by freben — APPROVED** [review]

**Pull request merged**